tools/ci/ci_wptrunner_infrastructure.sh

```diff
@@ -6,22 +6,22 @@ WPT_ROOT=$SCRIPT_DIR/../..
 cd $WPT_ROOT
 
 test_infrastructure() {
-    PY3_FLAG="$2"
-    TERM=dumb ./wpt $PY3_FLAG run --log-mach - --yes --manifest ~/meta/MANIFEST.json --metadata infrastructure/metadata/ --install-fonts --install-webdriver $1 $PRODUCT infrastructure/
+    PY2_FLAG="$2"
+    TERM=dumb ./wpt $PY2_FLAG run --log-mach - --yes --manifest ~/meta/MANIFEST.json --metadata infrastructure/metadata/ --install-fonts --install-webdriver $1 $PRODUCT infrastructure/
 }
 
 main() {
-    if [[ $# -eq 1 && "$1" = "--py3" ]]; then
-        PRODUCTS=( "chrome" )
-    else
+    if [[ $# -eq 1 && "$1" = "--py2" ]]; then
```

Comment:
There's another PR to make this always both; could we fold that change in here or would that make it harder to land?

Reply:
Yeah, seems reasonable; done.